### ISSUE TITLE
Add Tensorboard to Pretrained Model

### ISSUE THREAD

**Issue by Nada119** [mourra950/Siemens-System-Level-Modelling-of-ASDLA-Graduation-Project#61]
### What feature to add

We needed to add tensorboard feature to pretrained models

### Milestone

Milestone 2

### What labels is that issue related to

Pytorch, Jinja, Enhancement

**Comment by mourra950** [comment]
added tensorboard scripts inside the pretrained model template
